src/main/java/org/reaktivity/nukleus/tcp/internal/writer/stream/StreamFactory.java

```diff
@@ -154,6 +158,7 @@ private void processData(
                 slot = writeSlab.written(id, slot, writeBuffer);
                 if (slot == OUT_OF_MEMORY)
                 {
+                    streamsResetPartialWrite.increment();
```

Comment:
We tend not to pass the entire `AtomicCounter`, and instead prefer to pass a method reference to increment (and return) the count of overflowed streams due to partial writes.

For example, in `Writer`:
```java
this.incrementOverflow = context.counters().streamsOverflowed()::increment;
```